src/RegExLib.Web/Endpoints/Expressions/List.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Ardalis.ApiEndpoints;
+using RegExLib.Core.Entities;
+using RegExLib.SharedKernel.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RegExLib.Web.Endpoints.Expressions
+{
+    public class List : BaseAsyncEndpoint<List<ExpressionResponse>>
+    {
+        private readonly IRepository _repository;
+
+        public List(IRepository repository)
+        {
+            _repository = repository;
+        }
+
+        [HttpGet("/Expressions")]
+        [SwaggerOperation(
+            Summary = "Gets a list of all Expressions",
+            Description = "Gets a list of all Expressions",
+            OperationId = "Expression.List",
+            Tags = new[] { "ExpressionEndpoints" })
+        ]
+        public override async Task<ActionResult<List<ExpressionResponse>>> HandleAsync()
+        {
+            var expressions = (await _repository.ListAsync<Expression>())
+                .Select(expression => new ExpressionResponse(expression.Id, expression.Title, expression.Pattern, expression.Description, expression.AuthorId));
+
+            return Ok(expressions);
+        }
+    }
+}
```

Comment:
It looks like Steve added an .editorconfig to enforce formatting rules but this file doesn't conform. There might be others. I didn't check them all.

Reply:
any file I work in I apply formatting rules.maybe this befoure i get latest.I will review all and apply.

Reply:
Done [a40c02f](https://github.com/ardalis/RegExLib.com/pull/25/commits/a40c02fe19ad2b3bed0677f78aa3a41b2b36fba5)